src/providers/pronouns/alejo/PronounsAlejoApi.cpp

```diff
@@ -0,0 +1,124 @@
+#include "providers/pronouns/alejo/PronounsAlejoApi.hpp"
+
+#include "common/network/NetworkRequest.hpp"
+#include "common/network/NetworkResult.hpp"
+#include "common/QLogging.hpp"
+#include "providers/pronouns/UserPronouns.hpp"
+
+namespace chatterino::pronouns {
+
+UserPronouns AlejoApi::parse(const QJsonObject &object)
```

Comment:
nit: move this function to below the `fetch` function so it's consistent with the header order